### PR TITLE
Add article

### PR DESCRIPTION
**Pull request by caroleolivier**

### CODE REVIEW

**Comment by caroleolivier** [comment]
@simonproffitt : hi! can you review my article? It is super short and I think you will find it useful :)

File: _posts/2017-10-03-continuous-integration-in-github.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Done!
+
+I now have a CI tool set up for my [project](https://github.com/caroleolivier/minimal-react-starter/tree/v3.0.0) and can monitor the status on the [Build Status page](https://travis-ci.org/caroleolivier/minimal-react-starter) (as well as being notified automatically by email) so cool!
+
+
+#### Worth knowing about Travis
+
+_Build lifecycle_
+
+I found that [documentation page](https://docs.travis-ci.com/user/customizing-the-build#The-Build-Lifecycle) very useful to understand what Travis is doing when it builds and tests. This will certainly become handy later when/if my build becomes more complex.
```
Comment (simonproffitt) [inline _posts/2017-10-03-continuous-integration-in-github.md]: Semi colon before "very" ?

**Review by simonproffitt — CHANGES_REQUESTED** [review]
Just that single grammatical point, otherwise another cool article :)

File: _posts/2017-10-03-continuous-integration-in-github.md (hunk 1 above)
Comment (caroleolivier) [reply _posts/2017-10-03-continuous-integration-in-github.md]: I have rephrased actually.

File: _posts/2017-10-03-continuous-integration-in-github.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+date: 2017-10-03
+---
+
+I have been working on a project for the last couple of days and I decided it was about time I set up some kind of Continuous Integration tool to my repo in GitHub. I will describe here how you can achieve that in a few minutes with Travis CI.
+
+First of all, what is Continuous Integration?
+
+> Continuous integration is a DevOps software development practice where developers regularly merge their code changes into a central repository, after which automated builds and tests are run. Continuous integration most often refers to the build or integration stage of the software release process and entails both an automation component (e.g. a CI or build service) and a cultural component (e.g. learning to integrate frequently). The key goals of continuous integration are to find and address bugs quicker, improve software quality, and reduce the time it takes to validate and release new software updates.
+<br />
+> [AWS](https://aws.amazon.com/devops/continuous-integration/)
```
Comment (simonproffitt) [inline _posts/2017-10-03-continuous-integration-in-github.md:13]: Is this right? The '>' means the link/text doesn't render properly

File: _posts/2017-10-03-continuous-integration-in-github.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Done!
+
+I now have a CI tool set up for my [project](https://github.com/caroleolivier/minimal-react-starter/tree/v3.0.0) and can monitor the status on the [Build Status page](https://travis-ci.org/caroleolivier/minimal-react-starter) (as well as being notified automatically by email) so cool!
+
+
+#### Worth knowing about Travis
+
+_Build lifecycle_
+
+I found that [documentation page](https://docs.travis-ci.com/user/customizing-the-build#The-Build-Lifecycle) about Travis build lifecycle. I think it is worth reading to get an idea of what happened at build time.
```
Comment (simonproffitt) [inline _posts/2017-10-03-continuous-integration-in-github.md]: "I found _this_ " (sorry, I missed this the first time)

**Comment by simonproffitt** [comment]
Sorry, I missed a couple of things but have added comments

File: _posts/2017-10-03-continuous-integration-in-github.md (hunk 2 above)
Comment (caroleolivier) [reply _posts/2017-10-03-continuous-integration-in-github.md:13]: weird, works when I run it locally. Hmm... Ignore it for now pls.

**Comment by caroleolivier** [comment]
I should start paying you for every that that needs changing to this...

**Review by simonproffitt — APPROVED** [review]
Looks good to me :)

**Comment by caroleolivier** [comment]
Thank you!

**Pull request merged**